modules/govuk/manifests/apps/ckan.pp

```diff
@@ -64,6 +64,9 @@
 ) {
   $ckan_home = '/var/ckan'
   $ckan_ini  = "${ckan_home}/ckan.ini"
+  $pycsw_config = "${ckan_home}/pycsw.cfg"
+
+  $pycsw_port = "${$port + 1}"
```

Comment:
Unless there's a specific reason for the PyCSW port to be set to be one higher than the CKAN port, I'd prefer if this was passed in as a separate parameter. That way, if you're wondering where port number 3221 is used, or wondering if there is anything using port 3221, searching for the number in the govuk-puppet repository should give you the answer.

Reply:
Seems reasonable, I've pushed up an amend for this.